pubsub/v2/iterator_test.go

```diff
@@ -34,14 +34,15 @@ import (
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/credentials/insecure"
 	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/types/known/fieldmaskpb"
 )
 
 var (
 	projName                = "P"
-	topicName               = "some-topic"
-	subName                 = "some-sub"
-	fullyQualifiedTopicName = fmt.Sprintf("projects/%s/topics/%s", projName, topicName)
-	fullyQualifiedSubName   = fmt.Sprintf("projects/%s/subscriptions/%s", projName, subName)
+	topicID                 = "some-topic"
+	subID                   = "some-sub"
+	fullyQualifiedTopicName = fmt.Sprintf("projects/%s/topics/%s", projName, topicID)
```

Comment:
to consider: there's a lot of these string builders in the test code.  Did you want to switch over to using the generated path functions from the generator?  e.g. https://github.com/googleapis/google-cloud-go/blob/main/pubsub/apiv1/path_funcs.go